webapp/components/form/AutocompleteDialog/store/actions/useOnInputFieldKeyDown.js

```diff
@@ -4,7 +4,8 @@ import { State } from '../state'
 
 export const useOnInputFieldKeyDown = ({ onClose, focusItem }) =>
   useCallback(
-    ({ list, state }) => (event) => {
+    ({ state }) => (event) => {
+      const list = State.getList(state)
       const items = State.getItems(state)
       const inputField = State.getInputField(state)
       switch (event.keyCode) {
```

Comment:
keyCode is deprecated. use key instead (https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode)